src/ui/templates/partials/insurance/credit-period-with-buyer-description.njk

```diff
@@ -0,0 +1,21 @@
+{% set contentStrings = CONTENT_STRINGS.CREDIT_PERIOD_WITH_BUYER %}
+
+{% set descriptionHtml %}
+  <p class="govuk-body" data-cy="credit-period-with-buyer-description-protects-you">{{ contentStrings.PROTECTS_YOU }}</p>
+  <p class="govuk-body" data-cy="credit-period-with-buyer-description-insures-you">{{ contentStrings.INSURES_YOU }}</p>
+  <p class="govuk-body" data-cy="credit-period-with-buyer-description-happens-before">{{ contentStrings.HAPPENS_BEFORE }}</p>
+{% endset %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters-from-desktop">
+
+    {{ govukDetails({
+      summaryText: contentStrings.INTRO,
+      html: descriptionHtml,
+      attributes: {
+        "data-cy": "credit-period-with-buyer-description"
```

Comment:
can the `credit-period-with-buyer` come from a fieldId?

Reply:
Updated - the only downside to this is that if this partial is used in another place (like we do with other partials), we may need to revert back as a relevant field ID maybe not be available.